google/data_source_google_compute_instance_group_test.go

```diff
@@ -193,14 +193,19 @@ func testAccCheckDataSourceGoogleComputeInstanceGroup(dataSourceName string) res
 
 func testAccCheckDataSourceGoogleComputeInstanceGroupConfig() string {
 	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+       family  = "debian-9"
```

Comment:
Would you mind fixing the indentation so it's consistent with the other resources in this file?